### PR TITLE
Update nncf export

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
## 📝 Description

- Provide a clear summary of the changes and the issue that has been addressed.
- 🛠️ Fixes # (issue number)

## Testing

```python
datamodule = MVTec(root="../datasets/MVTec")
datamodule.setup(stage="test")

export_path = "../results/weights/openvino/model.xml"
metadata = "../results/weights/openvino/metadata.json"
inferencer = OpenVINOInferencer(export_path, metadata=metadata)

image_metrics, pixel_metrics = [AUROC(), F1Score()], AUROC()

datamodule.eval_batch_size = 1
for batch in datamodule.test_dataloader():
	result:ImageResult = inferencer(batch['image_path'][0])
	for metrics in image_metrics:
		metrics.update(torch.tensor(result.pred_score, dtype=torch.float).unsqueeze(0), batch["label"].int())
		pred_mask = cv2.resize(result.pred_mask.squeeze(), [*batch["mask"].squeeze().shape])
		pred_mask = torch.tensor(pred_mask).to(torch.bool)
		pixel_metrics.update(pred_mask/255., batch["mask"].int().squeeze())

for metric in image_metrics:
	print(f"{metric.__class__.__name__}: {metric.compute().item()}")
print(f"{pixel_metrics.__class__.__name__}: {pixel_metrics.compute().item()}")
```

## ✨ Changes

Select what type of change your PR is:

- [x] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔨 Refactor (non-breaking change which refactors the code base)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📚 Documentation update
- [ ] 🔒 Security update

## ✅ Checklist

Before you submit your pull request, please make sure you have completed the following steps:

- [ ] 📋 I have summarized my changes in the [CHANGELOG](https://github.com/openvinotoolkit/anomalib/blob/main/CHANGELOG.md) and followed the guidelines for my type of change (skip for minor changes, documentation updates, and test enhancements).
- [ ] 📚 I have made the necessary updates to the documentation (if applicable).
- [ ] 🧪 I have written tests that support my changes and prove that my fix is effective or my feature works (if applicable).

For more information about code review checklists, see the [Code Review Checklist](https://github.com/openvinotoolkit/anomalib/blob/main/docs/source/markdown/guides/developer/code_review_checklist.md).

### CODE REVIEW

File: src/anomalib/models/components/base/export_mixin.py
Comment (samet-akcay) [inline src/anomalib/models/components/base/export_mixin.py:1]: Can you explain why this change is needed? It might be an idea to add it as a comment for a future reference

Comment (ashwinvaidya17) [reply src/anomalib/models/components/base/export_mixin.py:1]: I got better performance with this change. However, I feel @adrianboguszewski or @alexsu52 probably knows better.

**Review by adrianboguszewski — APPROVED** [review]

File: src/anomalib/models/components/base/export_mixin.py
Comment (adrianboguszewski) [reply src/anomalib/models/components/base/export_mixin.py:1]: Usually, it's better to use a validation set for calibration. I used the train set initially, as there were not enough images in the val set.

**Review by samet-akcay — APPROVED** [review]
